CMakeLists.txt

```diff
@@ -116,6 +116,19 @@ else()
     message(STATUS "HpsJava is disabled")
 endif()
 
+# Install Pede
+option(HPSMC_ENABLE_PEDE "Enable (and install if necessary) pede" OFF)
```

Comment:
Can you add this option to the build.sh file? We have been trying to keep all the build options there so it is a bit more transparent what options are available.